### ISSUE TITLE
Fix links and right section missmatch

### ISSUE THREAD

**Issue by andreivladbrg** [sablier-labs/docs#193]
If you go to either of these links:

1. https://docs.sablier.com/concepts/protocol/stream-types#unlock-linear
2. https://docs.sablier.com/concepts/protocol/stream-types#unlock-cliff

both will direct you to the `Exponential` or Cliff `Exponential` shape instead

| Unlock Linear  | Unlock Cliff                     |
|-----------------------------------------|----------------------------------------|
| ![Unlock Linear Shape](https://github.com/user-attachments/assets/01a55b2d-2b54-4c33-9434-fe638080897f) | ![Unlock Cliff Shape](https://github.com/user-attachments/assets/928b2560-8a66-440f-bf36-3b2c8c9e0bce) |

The issue affects all links, as each one simply directs you to the shape above with a level.
I’m not sure what causes this or how to fix it.

**Comment by PaulRBerg** [comment]
Huh. Interesting.

It might be a bug in Docusaurus. Upgrading to the last version might fix this.

**Comment by smol-ninja** [comment]
This issue is linked with the load time of the function plots. Since plots are generated after the page has been loaded, this leads to the page to reset itself, and move the view either up or down.

To fix this, what if we replace the dynamically generated plots with their `.webp` images? For example, if you visit https://docs.sablier.com/guides/lockup/etherscan#step-1-go-to-contract-page-1, you'll notice that images load very quickly, and you land on the expected section.

**Comment by andreivladbrg** [comment]
> This issue is linked with the load time of the function plot

nice finding

> what if we replace the dynamically generated plots with their .webp images

it would fix this, but doing so would make it harder to make changes as we would need to (i) update the plot code (ii) take a screenshot of it and (iii) upload the new screenshot plot.

if there is no other fix to keep the current method, i guess we should go for images

**Comment by smol-ninja** [comment]
> it would fix this, but doing so would make it harder to make changes as we would need to (i) update the plot code (ii) take a screenshot of it and (iii) upload the new screenshot plot.

I agree but its a one time job per release. Also, we should still keep the plot code somewhere in the docs to reuse it but avoid publishing it directly on the stream shape page.

**Comment by PaulRBerg** [comment]
in Docusaurus, it is possible to have [`unlisted` documents](https://docusaurus.io/docs/next/api/plugins/@docusaurus/plugin-content-blog#markdown-front-matter).
